container/docker/handler.go

```diff
@@ -135,18 +135,18 @@ func newDockerContainerHandler(
 	id := ContainerNameToDockerId(name)
 
 	// Add the Containers dir where the log files are stored.
-	otherStorageDir := path.Join(path.Dir(storageDriverDir), pathToContainersDir, id)
+	otherStorageDir := path.Join(storageDir, pathToContainersDir, id)
```

Comment:
not in this PR, but add a TODO to give more descriptive names than storageDir and otherStorageDir